src/main/java/uk/gov/digital/ho/proving/income/audit/statistics/PassStatisticsAccumulator.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.digital.ho.proving.income.audit.statistics;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+import uk.gov.digital.ho.proving.income.audit.AuditResultByNino;
+import uk.gov.digital.ho.proving.income.audit.AuditResultType;
+import uk.gov.digital.ho.proving.income.audit.AuditResultTypeComparator;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
+class PassStatisticsAccumulator {
+
+    private final LocalDate fromDate;
+    private final LocalDate toDate;
+
+    private final Map<String, BestResult> bestResultByNino;
+    private final AuditResultTypeComparator resultTypeComparator;
```

Comment:
Comparator seems overkill unless you ever have a collection of the compared type.